crates/spirv-std/src/float.rs

```diff
@@ -0,0 +1,12 @@
+/// Abstract trait representing a SPIR-V floating point type.
+pub trait Float: num_traits::Float + crate::scalar::Scalar + Default {
```

Comment:
Shouldn't this be sealed and/or `unsafe`? (Same concern as with `Vector`)

Reply:
Sealed is implied by `Scalar`, since implementing `Scalar` requires `Sealed`, so does implementing `Float`.